ckanext/pages/theme/templates_main/ckanext_pages/blog_list.html

```diff
@@ -2,7 +2,7 @@
 
 {% block page_primary_action %}
   {% if h.check_access('ckanext_pages_update', {}) %}
-    {% link_for _('Add Article'), controller='ckanext.pages.controller:PagesController', action='blog_edit', page='', class_='btn btn-primary', icon='plus-sign-alt' %}
+    {% link_for _('Add Article'), controller='ckanext.pages.controller:PagesController', action='blog_edit', page='', class_='btn btn-primary', icon='plus-square' %}
```

Comment:
In that case I'd prefer to put these changes into version-dependent conditionals.